lib/rubocop/chef/cookbook_helpers.rb

```diff
@@ -48,6 +59,27 @@ def method_arg_ast_to_string(ast)
 
       private
 
+      #
+      # given a node object does it look like a chef resource or not?
+      #
+      # @param [RuboCop::AST::Node] node AST object to test
+      #
+      # @return [boolean]
+      #
+      def looks_like_resource?(node)
+        return false unless node.block_type? # resources are blocks if they have properties
```

Comment:
but what about `package "foo"`?  or is the point mostly to go through the properties so you don't really care?